src/plugins/mpris/command.cpp

```diff
@@ -0,0 +1,138 @@
+// albert extension mpris - a mpris interface plugin for albert
+// Copyright (C) 2016 Martin Buergmann
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "command.h"
+#include <QtDBus/QDBusConnection>
+#include <QtDBus/QDBusMessage>
+#include <QtDBus/QDBusInterface>
+#include <QDebug>
+#include "albertapp.h"
+#include "item.h"
+
+namespace MPRIS {
+
+Command::Command(QString &label, QString &title, QString &method, QString &iconpath)
+    : label_(label), title_(title), method_(method), iconpath_(iconpath), closeOnEnter_(false)
+{
+//    fireCallback([](){});
+}
+
+Command::Command(const char *label, const char *title, const char *method, QString iconpath)
+    : label_(label), title_(title), method_(method), iconpath_(iconpath), closeOnEnter_(false)
+{
+//    fireCallback([](){});
+}
+
+QString& Command::getIconPath() {
```

Comment:
please use 3 newlines and javadoc comments /** ****...***/ to seperate funcion bodies in cpp files